### PR TITLE
sci-mathematics/normaliz: version bump to 3.8.9

### PR DESCRIPTION
**Pull request by epsilon-0**
Package-Manager: Portage-3.0.8, Repoman-3.0.1
Signed-off-by: Aisha Tammy <gentoo@aisha.cc>

### CODE REVIEW

File: sci-mathematics/normaliz/normaliz-3.8.9.ebuild [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+# would be nice to package scip and cocoalib
+
+RDEPEND="
+	dev-libs/gmp:=[cxx]
+"
+DEPEND="${RDEPEND}
+	dev-libs/boost
+"
+# Only a boost header is needed -> not RDEPEND
```
Comment (DarthGandalf) [inline sci-mathematics/normaliz/normaliz-3.8.9.ebuild:27]: Should it use := to use the new header when new boost is installed?

Comment (epsilon-0) [reply sci-mathematics/normaliz/normaliz-3.8.9.ebuild:27]: I'm not even sure but you are probably right.
Just let this sit for a few days, my brain is fried.
Dumb package broke backwards compatibility so I'm not even sure if we want to add a new subslot or not...

File: sci-mathematics/normaliz/normaliz-3.8.9.ebuild [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+MYP="Normaliz-${PV}"
+
+DESCRIPTION="Tool for computations in affine monoids and more"
+HOMEPAGE="http://www.mathematik.uni-osnabrueck.de/normaliz/"
+SRC_URI="https://github.com/Normaliz/Normaliz/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0/3"
+KEYWORDS="~amd64 ~arm ~x86 ~amd64-linux ~x86-linux"
+IUSE="doc extras openmp static-libs"
```
Comment (SoapGentoo) [inline sci-mathematics/normaliz/normaliz-3.8.9.ebuild:17]: remove `static-libs` pls

File: sci-mathematics/normaliz/normaliz-3.8.9.ebuild (hunk 1 above)
Comment (SoapGentoo) [reply sci-mathematics/normaliz/normaliz-3.8.9.ebuild:27]: @DarthGandalf from my quick glance, it only uses C++ headers, and doesn't link, so this should be fine

File: sci-mathematics/normaliz/normaliz-3.8.9.ebuild [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+src_prepare() {
+	default
+	eautoreconf
+}
+
+src_configure() {
+	econf \
+		$(use_enable openmp) \
+		$(use_enable static-libs static)
```
Comment (SoapGentoo) [inline sci-mathematics/normaliz/normaliz-3.8.9.ebuild:43]: `--disable-static`

**Review by SoapGentoo — CHANGES_REQUESTED** [review]

File: sci-mathematics/normaliz/normaliz-3.8.9.ebuild (hunk 2 above)
Comment (epsilon-0) [reply sci-mathematics/normaliz/normaliz-3.8.9.ebuild]: done

File: sci-mathematics/normaliz/normaliz-3.8.9.ebuild (hunk 3 above)
Comment (epsilon-0) [reply sci-mathematics/normaliz/normaliz-3.8.9.ebuild]: done

**Pull request closed without merging**